sdk/python/feast/cli.py

```diff
@@ -47,6 +47,11 @@
 from feast.utils import maybe_local_tz
 
 _logger = logging.getLogger(__name__)
+tagsOption = click.option(
+    "--tags",
+    help="Filter by tags (e.g. 'key:value, key:value, ...')",
+    multiple=True,
```

Comment:
good catch! my original intention was to also support the passing of multiple `--tags` options, but i was hitting the following issue as described in the docs you linked and i forgot to remove that setting.
_When passing a default with `multiple=True`, the default value must be a list or tuple, otherwise it will be interpreted as a list of single characters._

Those docs have gotten me past that hurdle and we can now support the passing of multiple tags AND/OR a singular comma-separated list. If both are used, the dictionary is updated accordingly. For example - 
```bash
$ feast feature-views list --tags "team : driver_performance, other:tag" --tags missingtag: --tags "other:two" --tags other:3 
```
would become the following dict -
```python
{'team': 'driver_performance', 'other': '3', 'missingtag': ''}
```
i like this approach because it affords flexibility to the user and we have to massage the string inputs anyway, so there's no harm in supporting a comma-separated list as well.

Reply:
thoughts?

Reply:
fine for me, @tokoko ?